hack/generated/go.mod

```diff
@@ -79,4 +80,4 @@ require (
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
 
-replace github.com/Azure/azure-service-operator => ../../
+replace github.com/Azure/azure-service-operator/v2 => ../../v2
```

Comment:
(Or something else?)

Reply:
Neither, stuff has moved and it wouldn't work without the replace, /shrug

The version here seems to be fake, for local replaced modules? But on the other hand the entry for _this_ module in the v2 `go.mod` _does_ have a real version. I have no idea why.

Reply:
yeah I thought that this was so that you got stuff from the local environment rather than it trying to actually put what's hosted on GitHub?

Reply:
> yeah I thought that this was so that you got stuff from the local environment rather than it trying to actually put what's hosted on GitHub?

That's what I thought too. I don't know why it added a real version in the other direction.